src/main/java/messenger/ImageMessengerRequestProcessor.java

```diff
@@ -0,0 +1,42 @@
+package messenger;
+
+import notification.Notification;
+import request.RequestProcessor;
+
+import org.springframework.web.multipart.MultipartFile;
+import org.springframework.web.multipart.MultipartHttpServletRequest;
+
+import javax.servlet.http.HttpServletRequest;
+
+/**
+ * Created by annterina on 11.11.17.
+ */
+public class ImageMessengerRequestProcessor extends RequestProcessor {
+
+    private MultipartFile image;
+
+    @Override
+    protected void preprocess(HttpServletRequest request)
+    {
+        super.preprocess(request);
+        MultipartHttpServletRequest multipartRequest = (MultipartHttpServletRequest) request;
+        image = multipartRequest.getFile("image");
+    }
+
+    @Override
+    protected void postprocess(HttpServletRequest request, Notification notification)
+    {
+        super.postprocess(request, notification);
```

Comment:
This is rather unnecessary as `postprocess(request, notification)` from `RequestProcessor` class still will be executed if not overridden and this does the same.

Reply:
Yeah, but for me it is still kind of example how to use our library. There will be no difference after removing that part, that's true, but for me it demonstrates usages of the library in better way.